table/shell/src/main/java/alluxio/cli/table/command/LoadTableCommand.java

```diff
@@ -0,0 +1,343 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.table.command;
+
+import alluxio.AlluxioURI;
+import alluxio.annotation.PublicApi;
+import alluxio.cli.CommandUtils;
+import alluxio.cli.fs.FileSystemShellUtils;
+import alluxio.cli.fs.command.AbstractDistributedJobCommand;
+import alluxio.cli.fs.command.DistributedLoadUtils;
+import alluxio.client.file.FileSystemContext;
+import alluxio.client.file.URIStatus;
+import alluxio.client.table.TableMasterClient;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.PropertyKey;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.status.AlluxioStatusException;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.table.common.CatalogPathUtils;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Loads table into Alluxio space, makes it resident in memory.
+ */
+@ThreadSafe
+@PublicApi
+public class LoadTableCommand extends AbstractTableCommand {
+  private static final int DEFAULT_REPLICATION = 1;
```

Comment:
why we need this configuration? can we simply delegate the default replication value?

Reply:
There is no default replication configuration. 
For now, there are 3 configurations about replication:

- `alluxio.user.file.replication.max`
- `alluxio.user.file.replication.min`
- `alluxio.user.file.replication.durable`

Configurations above are not suitable in this scene.